sdk/core/Microsoft.Extensions.Azure/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure client integration for ASP.NET Core
+
+Microsoft.Extensions.Azure.Core provides shared primitives to integrate Azure clients with ASP.NET Core dependency injection and configuration systems.
+
+[Source code][source_root] | [Package (NuGet)][package]
+
+## Installing
+
+Install the ASP.NET Core integration library using [NuGet][nuget]:
+
+```
+dotnet add package Microsoft.Extensions.Azure
+```
+
+## Usage Scenarios and Samples
+
+Use add `AddAzureClients` call to you `ConfigureServices` method. You can use provided builder to register client instances with dependency injection container.
+
+```C# Snippet:ConfigureServices
+public void ConfigureServices(IServiceCollection services)
+{
+    // Registering policy to use in ConfigureDefaults later
+    services.AddSingleton<DependencyInjectionEnabledPolicy>();
+
+    services.AddAzureClients(builder => {
+
+        builder.AddSecretClient(Configuration.GetSection("KeyVault"))
```

Comment:
Nit - Add a comment like "// Configure a KeyVault SecretClient using ..." to help add context.  If I wasn't familiar with KeyVault, I might think AddSecretClient was doing some random DI thing I had never heard of before.